external/webserver/http_client.c

```diff
@@ -740,3 +833,93 @@ int http_send_response(struct http_client_t *client, int status, const char *bod
 	HTTP_FREE(buf);
 	return HTTP_OK;
 }
+
+int http_send_response_with_status_msg(struct http_client_t *client, int status, const char* status_message, const char* body, struct http_keyvalue_list_t *headers)
```

Comment:
If possible you can implement this function with curl. then I think you can implement below features easier.

Reply:
...